HW0/P4/P4.py

```diff
@@ -52,11 +79,48 @@
     # a = ?
     # s = ?
 
+    #make a 6 x K matrix to store
+    S = np.zeros([6,K])
+    S = np.matrix(S)
+
+    # make A
+    A = np.zeros([6,6])
+    A = np.matrix(A)
+    A[0,0] = 1
```

Comment:
thanks fanny! i'll try to implement a for loop and look for the patterns next time!